docs/operating-scylla/nodetool-commands/tasks-wait.rst

```diff
@@ -0,0 +1,48 @@
+Nodetool tasks-wait
+===================
+**tasks-wait** - Wait until a task manager task is finished and gets its status.
+
+Syntax
+-------
+.. code-block:: console
+
+   nodetool tasks-wait <task_id>
+
+For example:
+
+.. code-block:: shell
+
+   > nodetool tasks-wait ef1b7a61-66c8-494c-bb03-6f65724e6eee
+
+Example output
+--------------
+
+.. code-block:: shell
+
+    id : ef1b7a61-66c8-494c-bb03-6f65724e6eee
+    type : repair
+    scope : keyspace
+    state : done
+    is_abortable : true
+    start_time : 2024-07-02T12:10:43Z
+    end_time : 2024-07-02T12:10:43Z
+    error :
+    parent_id : 00000000-0000-0000-0000-000000000000
+    sequence_number : 1
+    shard : 0
+    keyspace : abc
+    table :
+    entity :
+    progress_units : ranges
+    progress_total : 8
+    progress_completed : 8
+    children_ids : [1b0affc8-b8f5-4324-a0f7-23616e9d7ef5, 9dd72d59-6dcf-48d6-bf88-82936a3aac4e]
```

Comment:
Let's have a --quiet variant that doesn't print anything, but returns 0 exit code if the task was successful, 1 if it failed, 2 if waiting timed out (add a --timeout parameter; useful if the script waiting for it wants to do other things meanwhile) and 125 if an error occurred.